parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -219,8 +218,21 @@ loop = do
       getHQ'Terms p = BranchUtil.getTerm (resolveSplit' p) root0
       getHQ'Types :: Path.HQSplit' -> Set Reference
       getHQ'Types p = BranchUtil.getType (resolveSplit' p) root0
+      getHQTerms :: HQ.HashQualified -> Action' m v (Set Referent) 
```

Comment:
Handy utility function which we thought should exist, but didn't yet. Converts a hash qualified to a `Set Referent`, taking into account current path and branch.